galasa-inttests-parent/dev.galasa.inttests/src/main/java/dev/galasa/inttests/docker/local/DockerLocalJava08Ubuntu.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
+package dev.galasa.inttests.docker.local;
+
+import dev.galasa.BeforeClass;
+import dev.galasa.Test;
+import dev.galasa.TestAreas;
+import dev.galasa.galasaecosystem.IGenericEcosystem;
+import dev.galasa.galasaecosystem.ILocalEcosystem;
+import dev.galasa.galasaecosystem.LocalEcosystem;
+import dev.galasa.inttests.docker.AbstractDockerLocal;
+import dev.galasa.ipnetwork.ICommandShell;
+import dev.galasa.java.JavaVersion;
+import dev.galasa.java.ubuntu.IJavaUbuntuInstallation;
+import dev.galasa.java.ubuntu.JavaUbuntuInstallation;
+import dev.galasa.linux.ILinuxImage;
+import dev.galasa.linux.LinuxImage;
+import dev.galasa.linux.OperatingSystem;
+
+@Test
+@TestAreas({"dockermanager", "localecosystem", "java08", "ubuntu"})
+public class DockerLocalJava08Ubuntu extends AbstractDockerLocal {
+	
+	@LocalEcosystem(linuxImageTag = "PRIMARY")
+	public ILocalEcosystem ecosystem;
+	
+	@LinuxImage(operatingSystem = OperatingSystem.ubuntu)
+	public ILinuxImage linuxImage;
+	
+	@JavaUbuntuInstallation(javaVersion = JavaVersion.v8)
+	public IJavaUbuntuInstallation java;
+	
+	@BeforeClass
+	public void setProps() throws Exception {
+		ecosystem.setCpsProperty("docker.default.engines", "DKRENGINE01");
+		ecosystem.setCpsProperty("docker.engine.DKRENGINE01.hostname", "https://dkrengine01.cics-ts.hur.hdclab.intranet.ibm.com");
+		ecosystem.setCpsProperty("docker.engine.DKRENGINE01.port", "2376");
+		ecosystem.setCpsProperty("docker.engine.DKRENGINE01.max.slots", "3");
+	}
+	
+	public void getShell() throws Exception {
```

Comment:
Best to to a getDockerLinux() return the linux image, so it can be reused for other things